File: src/main/java/com/adjorno/billib/rest/db/SpotifyUrl.java
package com.adjorno.billib.rest.db;

import javax.persistence.*;

@Entity
@Table(name = "SPOTIFY_URL")
public class SpotifyUrl {
    @Id
    @Column(name = "TRACK_ID")
    private Long mTrackId;

    @Column(name = "SPOTIFY_URL")
    private String mSpotifyUrl;

    public Long getTrackId() {
        return mTrackId;
    }

    public void setTrackId(Long trackId) {
        mTrackId = trackId;
    }

    public String getSpotifyUrl() {
        return mSpotifyUrl;
    }

    public void setSpotifyUrl(String spotifyUrl) {
        mSpotifyUrl = spotifyUrl;
    }

    @Override
    public String toString() {
        return mTrackId + " - " + mSpotifyUrl;
    }
}

File: src/main/java/com/adjorno/billib/rest/db/SpotifyUrl.kt
package com.adjorno.billib.rest.db

import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "SPOTIFY_URL")
data class SpotifyUrl(
        @Id
        @Column(name = "TRACK_ID")
        var trackId: Long? = null,

        @Column(name = "SPOTIFY_URL")
        var spotifyUrl: String? = null
)
